portfolio/src/main/webapp/game.js

```diff
@@ -1,18 +1,19 @@
 var canvas = document.getElementById("game");
 var ctx = canvas.getContext("2d");
-var xPos = canvas.width / 2;
-var yPos = canvas.height / 2; 
 
-/* 2D array that stores current board state
-* 0 = empty, 1 = red, 2 = black
-*/
-var board = [
-  [0, 0, 0],
-  [0, 0, 0],
-  [0, 0, 0]
-];
+// is there some way to make this private?
```

Comment:
Following up from sync conversation, you could use an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) or a [class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes).